### PR TITLE
Add ft fb type 100 composbox cases

### PR DESCRIPTION
**Pull request by ShakoHo**
@askeing @MikeLien @ypwalter r?

### CODE REVIEW

**Comment by askeing** [comment]
```
./tests/regression/facebook/test_chrome_facebook_ft_type_composerbox_100_txt.sikuli/test_chrome_facebook_ft_type_composerbox_100_txt.py:40: E226 missing whitespace around arithmetic operator
./tests/regression/facebook/test_firefox_facebook_ft_type_composerbox_100_txt.sikuli/test_firefox_facebook_ft_type_composerbox_100_txt.py:40: E226 missing whitespace around arithmetic operator
```

File: tests/regression/facebook/test_chrome_facebook_ft_type_composerbox_100_txt.sikuli/test_chrome_facebook_ft_type_composerbox_100_txt.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+chrome = browser.Chrome()
+fb = facebook.facebook()
+
+chrome.clickBar()
+chrome.enterLink(sys.argv[3])
+fb.wait_for_loaded()
+
+sleep(2)
+setAutoWaitTimeout(10)
```
Comment (ypwalter) [inline tests/regression/facebook/test_chrome_facebook_ft_type_composerbox_100_txt.sikuli/test_chrome_facebook_ft_type_composerbox_100_txt.py]: `setAutoWaitTimeout()` is not useful if there is no `wait()`
Please refer to #639 for further information.

File: lib/sikuli/common.sikuli/common.py [hunk 2]
```diff
@@ -93,6 +93,9 @@ def is_infolog_enabled(self):
     def set_mouse_delay(self, sec=0.5):
         Settings.MoveMouseDelay = sec
```
Comment (ypwalter) [inline lib/sikuli/common.sikuli/common.py]: (This is not set_type_delay related comment.)
I found an interesting thing. Maybe we should also set Settings.DelayBeforeMouseDown to 0! Its default value is 300ms which matches exactly what we measured before for MouseDown event! @askeing @MikeLien @ShakoHo , how do you think?

Reference: http://sikulix-2014.readthedocs.io/en/latest/scripting.html

File: tests/regression/facebook/test_firefox_facebook_ft_type_composerbox_100_txt.sikuli/test_firefox_facebook_ft_type_composerbox_100_txt.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+capture_width = int(sys.argv[6])
+capture_height = int(sys.argv[7])
+
+com.set_type_delay(0.06)
+
+t1 = time.time()
+capimg1 = capture(0, 0, capture_width, capture_height)
+
+char_list = []
+for tmp_index in range(100):
```
Comment (ypwalter) [inline tests/regression/facebook/test_firefox_facebook_ft_type_composerbox_100_txt.sikuli/test_firefox_facebook_ft_type_composerbox_100_txt.py]: how about:
`
import string
(list(string.ascii_lowercase)*4)[:100]
`

**Comment by askeing** [comment]
lgtm, please squash commits before merge

**Pull request merged**